### DISCUSSION TITLE
骁龙845/855的设备可以通过特殊的方法刷入Windows on ARM，为何不适配ARM64

### DISCUSSION THREAD

**Discussion by i486-DX9** [Meloong-Git/PCL#4671] [提问] [answered]
**骁龙845/855的设备**可以通过特殊的方法刷入Windows on ARM（例如Mindows安装器），虽然野鸡但是至少能用，而且兼容性列表中还有小米平板5（骁龙860，是855的马甲）。想问一下龙猫为啥不适配PCL2 on ARM64，是因为BakaXL/HMCL等启动器还没适配，不必“皇上不急太监急”这种行为？还是因为自己没有买设备，也不想这么干？

**Comment by tangge233** [comment]
.NET Framework 4.6.2 并不支持原生 ARM64，如果要支持上原生 ARM64 就需要升级到 .NET。这工作量很大，并且[龙猫已经表过态暂时不会考虑](https://github.com/Hex-Dragon/PCL2/issues/4185#issuecomment-2212384475)

**Comment by allMagicNB** [comment]
你是太监？

**Comment by hejiehao** [comment]
HMCL 已经适配了，详情见 https://github.com/HMCL-dev/HMCL/blob/main/PLATFORM_cn.md

**Comment by LeiSureLyYrsc** [comment]
你本来可以去 Issue 和气地提出“PCL2 对于 WinArm”的支持，结果你非要说一句“皇上不急太监急”，难道你是开发者的 ~~Father~~ 吗？
如果你急着要的话可以考虑去爱发电给龙猫打钱来解决所有问题❤️

你说“还是因为自己没有买设备，也不想这么干？”
为什么开发者要自费买设备来开发？谁来出钱？

如果你有需要，请你去给 PCL2 实现兼容 WinArm 支持
Pull Request 欢迎你

**Comment by wyc-26** [comment]
不是你想要啥人家开发者就愿意做啥的
别当伸手党行吗。。。你要有技术你直接去交 PR
你也不考虑一下能不能实现的问题，评价为站着说话不腰疼

---

> 是因为BakaXL/HMCL等启动器还没适配，不必“皇上不急太监急”这种行为？

感觉你的意思就是，PCL 的代码基本上都是从另两个启动器里面抄的。。。
另两个启动器做不出来 PCL 就抄不了是吧 😅

> 还是因为自己没有买设备，也不想这么干？

虚拟机：？

**Comment by WForst-Breeze** [comment, marked as answer]
开发者并非在全职开发 PCL2，因此开发者需要将精力投入到更具有开发价值、能为绝大部分用户带来明显的使用体验提升的功能中去
